src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/OverloadMethodsDeclarationOrderCheck.java

```diff
@@ -51,14 +72,40 @@
  * public void notFoo() {}
  * </pre>
  * <p>
- * Example of incorrect grouping of overloaded methods:
+ * To configure the check to group static and private methods separately within their own
+ * individual groups
+ * </p>
+ * <pre>
+ * &lt;module name="OverloadMethodsDeclarationOrder"&gt;
+ *   &lt;property name="modifierGroups" value="static, private"&gt;&lt;/property&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example of code with violation
  * </p>
  * <pre>
  * public void foo(int i) {} // OK
  * public void foo(String s) {} // OK
- * public void notFoo() {} // violation. Have to be after foo(String s, int i)
- * public void foo(int i, String s) {}
- * public void foo(String s, int i) {}
+ * public void notFoo() {} // OK
+ * private void foo(boolean b) {} // OK
+ * protected void foo(String s, int i) {} // Violation. Has to be after foo(String s)
```

Comment:
please make all `// violation` comments consistent in capitalization and spacing. Also, please do `// violation, should be after 'foo(String s)'` to make examples as easy to read and understand as possible, for all cases.

Reply:
I can't combine the `should be after..` with https://github.com/checkstyle/checkstyle/pull/12086#discussion_r974851851 can I? Seems like I need to do one or the other

Reply:
disregard above comment